pkg/gameserverallocations/allocator.go

```diff
@@ -108,6 +107,7 @@ type Allocator struct {
 	remoteAllocationCallback     func(context.Context, string, grpc.DialOption, *pb.AllocationRequest) (*pb.AllocationResponse, error)
 	remoteAllocationTimeout      time.Duration
 	totalRemoteAllocationTimeout time.Duration
+	allocationBatchWaitTime      time.Duration
```

Comment:
```suggestion
	batchWaitTime      time.Duration
```

Nit: Seems redundant to say allocation again, since we're already in the allocator.